### DISCUSSION TITLE
I can't delete loading.tsx

### DISCUSSION THREAD

**Discussion by yusufakcekoce** [vercel/next.js#52207] [Help]
### Summary

I'm working on a project with Next.js I added loading.tsx inside the app directory, but when I try to delete it, I get an error;

```
Caused by: The system cannot find the file specified. (os error 2)

Import trace for requested module: ./src/app/loading.tsx
```

After deleting it, I am unable to bring up the application

**Comment by corn-flour** [comment]
I faced this issue before, what worked for me was to delete the `.next` folder and start the server again

**Reply by yusufakcekoce** [reply]
We have solved this error with remove the .next folder. But I can't understand why I'm having this problem

**Reply by corn-flour** [reply]
i'm guessing the dev build didn't accurately check for the deletion of that file to update accordingly, so it still expects the `loading.tsx` file to be where it was, but i don't know for sure

**Reply by yusufakcekoce** [reply]
I thought the same
